custom_components/tuya_local/devices/tuya_000004rime_weather_station.yaml

```diff
@@ -0,0 +1,674 @@
+name: Weather station
+products:
+  - id: nghjvn2pcjllwska
+    name: WeatherStation Pro™
+primary_entity:
+  entity: sensor
+  name: Local temperature
+  class: temperature
+  dps:
+    - id: 131
+      type: integer
+      name: sensor
+      unit: C
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: sensor
+    name: Weather icon
+    class: enum
+    icon: "mdi:weather-partly-snowy-rainy"
+    dps:
+      - id: 103
+        optional: true
+        type: base64
+        name: sensor
+        mapping:
+          - mask: "000000FF00000000000000000000000000"
+          - dps_val: 0
+            value: None
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 1
+            value: Sunny
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 2
+            value: Cloudy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 3
+            value: Sunny and cloudy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 4
+            value: Cloudy and light rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 5
+            value: Cloudy and moderate rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 6
+            value: Cloudy and heavy rain
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 7
+            value: Cloudy, rain and thunder
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 8
+            value: Snowy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 9
+            value: Cloudy and snowy
+            mask: 000000FF00000000000000000000000000
+          - dps_val: 10
+            value: Cloudy, rainy and snowy
+            mask: 000000FF00000000000000000000000000
+      - id: 103
+        optional: true
+        type: base64
+        name: Payload
```

Comment:
I don't think it is mentioned anywhere explicitly, but it seems to be a convention that attribute names are all lowercase.